### PR TITLE
Update background jobs selector documentation

### LINKED ISSUE #2353: no example how to select cron
although we recommend it.

https://doc.owncloud.com/server/10.4/admin_manual/configuration/server/occ_command.html#background-jobs-selector

### PR DESCRIPTION
**Pull request by settermjd**
This commit background jobs selector documentation to provide examples
of selecting each of the three schedulers, as well as links to other
relevant information in the documentation. This fixes #2353.

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/occ_command/background_jobs_selector.adoc [hunk 1]
```diff
@@ -11,18 +11,19 @@ background
  background:webcron    Use webcron to run background jobs
 ----
 
-This example selects Ajax:
+== Example
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/server/occ_command/background_jobs_selector.adoc]: Shouldnt it be plural as you have more than one example?

Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command/background_jobs_selector.adoc]: Right you are. Will update it.

**Comment by voroyam** [comment]
Thanks for this PR. It's maybe not urgent, but it adds value to the docs imho, providing the recommended background job example.

File: modules/admin_manual/pages/configuration/server/occ_command/background_jobs_selector.adoc (hunk 1 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command/background_jobs_selector.adoc]: Done!

**Pull request merged**